server/config/toml.go

```diff
@@ -44,6 +44,20 @@ halt-height = {{ .BaseConfig.HaltHeight }}
 # Note: Commitment of state will be attempted on the corresponding block.
 halt-time = {{ .BaseConfig.HaltTime }}
 
+# MinRetainBlocks defines the minimum block height offset from the current
+# block being committed, such that all blocks past this offset are pruned
+# from Tendermint. It is used as part of the process of determining the
+# ResponseCommit.RetainHeight value during ABCI Commit. A value of 0 indicates
+# that no blocks should be pruned.
+#
+# This configuration value is only responsible for pruning Tendermint blocks.
+# It has no bearing on application state pruning which is determined by the
+# "pruning-*" configurations.
+#
+# Note: This configuration can and should be used in conjunction with other
+# parameters to determine the correct minimum value of ResponseCommit.RetainHeight.
```

Comment:
or the server config, we probably should elaborate on this a bit - e.g. to say that the pruning is dependent on this parameter in combination with unbonding time, state sync snapshots, and state pruning, to make sure we don't remove blocks that the application needs.